src/blitter/32bpp_optimized.hpp

```diff
@@ -17,7 +17,7 @@ class Blitter_32bppOptimized : public Blitter_32bppSimple {
 public:
 	/** Data stored about a (single) sprite. */
 	struct SpriteData {
-		SpriteCollMap<uint32_t> offset[2]; ///< Offsets (from .data) to streams for different zoom levels, and the normal and remap image information.
+		std::array<SpriteCollMap<uint32_t>, 2> offset; ///< Offsets (from .data) to streams for different zoom levels, and the normal and remap image information.
```

Comment:
IMO either keep the C array here, or change it to `std::array` for all blitters.